app/src/main/res/raw/menu.json

```diff
@@ -155,6 +155,40 @@
 					}
 				]
 			}
+		},
+		{
+			"version": 2016072901,
+			"url": "https://moca.olografix.org/moca2016.ics",
+			"title": "MOCA 2016",
+			"start": "2016-07-19",
+			"end": "2016-07-21",
```

Comment:
Is that the right date?

(It's not a big deal since Giggity will update these in its database based on your .ics on first load, but still it'd look odd having your event listed as already in the past.)